spec/order_spec.rb

```diff
@@ -0,0 +1,41 @@
+
+require 'order'
+require 'menu'
+
+describe Order do 
+  subject(:order) { described_class.new(menu) }
+
+  let(:dishes) do 
+    { chicken: 1, chinese: 2 }
+  end 
+
+  let(:menu) { double(:menu) }
```

Comment:
As you are injecting a menu inside an Order class, while this is good, maybe for readability if should be placed next to the subject(:order)